### PR TITLE
Plenv plugin

### PR DESCRIPTION
**Pull request by ravenhall**
Initial creation of a simple plugin for plenv. See https://github.com/tokuhirom/plenv for details on plenv.

### CODE REVIEW

File: plugins/available/plenv.plugin.bash [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# plugin for plenv
+
+cite about-plugin
+about-plugin 'plenv plugin for Perl'
+
+pathmunge "${HOME}/.plenv/bin"
```
Comment (nwinkler) [inline plugins/available/plenv.plugin.bash]: Maybe wrap this in an `if` statement as well? Don't want to pollute the path if this doesn't exist...

File: plugins/available/plenv.plugin.bash [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+if [[ `which plenv` ]] ; then
+
+  # init plenv
+  eval "$(plenv init -)"
+
+  # Load the auto-completion script if it exists.
+  [[ -e ~/.plenv/completions/plenv.bash ]] && source ~/.plenv/completions/plenv.bash
+
+else echo "Unable to find plenv. See https://github.com/tokuhirom/plenv for installation"
```
Comment (nwinkler) [inline plugins/available/plenv.plugin.bash]: Not sure whether this is a good idea. We have had several reports where printing messages like this has caused issues when e.g. using Bash-it over SSH. If I remember correctly, `scp` fails if the remote server echoes a message like this during the login stage.

I prefer failing silently, i.e. nothing happens if you don't have plenv.

**Comment by nwinkler** [comment]
Thanks! I've added a couple of comments - can you please take a look?

**Comment by ravenhall** [comment]
I've updated the plugin as suggested. I'll close this pull request and open a new one.

**Pull request closed without merging**